zipline/data/bundles/core.py

```diff
@@ -586,8 +586,8 @@ def clean(name,
             def should_clean(name):
                 dt = from_bundle_ingest_dirname(name)
                 return (
-                    (before is not None and dt < before) or
-                    (after is not None and dt > after)
+                    (before is None or dt < before) and
+                    (after is None or dt > after)
```

Comment:
@llllllllll What is the intended way to clean out all the ingestions for a bundle? I tried with no args, but nothing happened. However, my change looks to fail the relevant test...

Reply:
You need to pass either `--before`, `--after` or `--keep-last` to specify which ingests you would like to delete.

Reply:
Ah, ok, that was my confusion. I expected that passing none of them would delete everything.